tls.tf

```diff
@@ -111,31 +111,26 @@ resource "tls_locally_signed_cert" "vault-server" {
 }
 
 # Encrypt server key with GCP KMS
-data "external" "vault-tls-key-encrypted" {
-  count = local.manage_tls_count
-
-  program = ["${path.module}/scripts/gcpkms-encrypt.sh"]
-
-  query = {
-    root     = path.module
-    data     = tls_private_key.vault-server[0].private_key_pem
-    project  = var.project_id
-    location = google_kms_key_ring.vault.location
-    keyring  = google_kms_key_ring.vault.name
-    key      = google_kms_crypto_key.vault-init.name
-  }
-
-  depends_on = [google_kms_crypto_key.vault-init]
+data "google_kms_secret_ciphertext" "vault-tls-key-encrypted" {
```

Comment:
There should be `count = local.manage_tls_count` as for all resources in this file.
Without it this module brokes with parameters:
```
  manage_tls          = false
  tls_save_ca_to_disk = false
```